docs/user-guide/api-mediation/certificate-configuration-scenarios.md

```diff
@@ -0,0 +1,252 @@
+# Certificate configuration scenarios
+
+As a system programmer, review the different scenarios for configuring Zowe for automatic certificate setup. Examples are provided for each scenario.
+
+Zowe servers require both a keystore to store the certificates as well as a truststore to validate of these certificates and any other certificates that are encountered.
+For use of Zowe on z/OS, keystore and truststore can either be unix file-based (PKCS12) or z/OS keyring-based. 
+Both the keystore and truststore can automatically be created by Zowe regardless of which storage type is used. Keystores and truststores can be populated either with certificates that the user chooses, or with self-signed certificates generated by Zowe. 
+This automation can be performed by defining and customizing the `zowe.setup.certificate` section of your Zowe YAML configuration.
+Zowe can then automate the certificate setup via the `zwe init certificate` command. This command is run as a subcommand of `zwe init`.
+
+**Note:**  
+Automated generation of certificates is an option, but is not required. If you already have a keystore that contains a valid certificate (see the description of a valid certificate), and the certificate's corresponding private key, along with a truststore which validates the certificate and any other certificates you expect to encounter, then you also have the option of directly defining the parameter `zowe.certificate` which specifies the location of each of these objects. Note that this parameter should not be confused with the parameter `zowe.setup.certificate`.
+
+## What is a valid certificate in Zowe?
+
+A valid certificate for use in Zowe can be either of the following formats:
```

Comment:
When you say "formats", I think file-based or keyring based.
The "formats" are as stated _above_. ("For use of Zowe on z/OS, keystore and truststore can either be unix file-based (PKCS12) or z/OS keyring-based.")
The _following_ bullets are what I'd think of as certificate attributes instead.
So you might say,

"A valid certificate for use in Zowe must have one of the following attributes, regardless of format:"

Reply:
Attributes sounds good.